### PR TITLE
Set WebView proxy to 0.0.0.0 when possible

### PR DESCRIPTION
**Pull request by link2xt**
requires https://github.com/deltachat/deltachat-core-rust/pull/4339 to be checked out at jni/deltachat-core-rust

### CODE REVIEW

**Comment by hpk42** [comment]
> **To test the changes in this pull request, install this apk:** [package app-preview.apk](https://github.com/deltachat/deltachat-android/suites/12339606636/artifacts/654843398)

this apk reliably works in my testing.

File: src/org/thoughtcrime/securesms/FullMsgActivity.java [hunk 1]
```diff
@@ -164,4 +167,33 @@ private void onChangeLoadRemoteContent(LoadRemoteContent loadRemoteContent) {
...
+        throw new Exception(dcContext.getLastError());
+      }
+
+      String mimeType = httpResponse.getMimetype();
+      if (mimeType == null) {
+        mimeType = "application/octet-stream";
+      }
+      String encoding = httpResponse.getEncoding();
+      if (encoding == null) {
+        encoding = "UTF-8";
```
[the remark this answers is not in the document]
Comment (r10s) [reply src/org/thoughtcrime/securesms/FullMsgActivity.java]: thanks for the hint, i changed that

**Comment by r10s** [comment]
> Release v1.112.8 of the core containing necessary API is ready.

thanks! i pulled that on android-master and rebased this pr, so there should be a fresh app-preview.apk soon.

**Review by r10s — APPROVED** [review]
i also checked setProxyOverride() docs, this lgtm :)

**Pull request merged**